projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/graphql/it/GraphQLJobExecutionsStatusCountIT.java

```diff
@@ -102,10 +102,10 @@ public void testExecutionStatusCount_expectTwoSuccessfulTwoFailed(String jobName
       var expectedEndTimeLarger = OffsetDateTime.now();
       var expectedEndTimeSmaller = OffsetDateTime.now().minusDays(1);
 
-      addJobExecution(expectedEndTimeLarger, "testId", ExecutionStatus.FINISHED, jobName);
-      addJobExecution(expectedEndTimeSmaller, "testId2", ExecutionStatus.FINISHED, jobName);
-      addJobExecution(expectedEndTimeLarger, "testI3", ExecutionStatus.FAILED, jobName);
-      addJobExecution(expectedEndTimeSmaller, "testId4", ExecutionStatus.FAILED, jobName);
+      addJobExecution(expectedEndTimeLarger, "testId", ExecutionStatus.SUCCEEDED, jobName);
+      addJobExecution(expectedEndTimeSmaller, "testId2", ExecutionStatus.SUCCEEDED, jobName);
+      addJobExecution(expectedEndTimeLarger, "testI3", ExecutionStatus.PLATFORM_ERROR, jobName);
+      addJobExecution(expectedEndTimeSmaller, "testId4", ExecutionStatus.PLATFORM_ERROR, jobName);
```

Comment:
Maybe introduce somewhere around here an execution with a USER_ERROR to test this use case.

Reply:
I added the PLATFORM_ERROR only to test that the API will work with status that contains underscore.